src/applications/accredited-representative-portal/containers/POARequestDetails.jsx

```diff
@@ -0,0 +1,173 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useParams, Link } from 'react-router-dom-v5-compat';
+import { formatDateShort } from 'platform/utilities/date/index';
+import usePOARequests from '../hooks/usePOARequests';
+
+const checkAuthorizations = (
+  isTreatmentDisclosureAuthorized,
+  isAddressChangingAuthorized,
+  status,
+) => {
+  const authorizations = [];
+  if (isTreatmentDisclosureAuthorized === status) {
+    authorizations.push('Health');
+  }
+
+  if (isAddressChangingAuthorized === status) {
+    authorizations.push('Address');
+  }
+  return authorizations.length > 0 ? authorizations.join(', ') : 'None';
+};
+
+const POARequestDetails = () => {
+  const { poaRequests } = usePOARequests();
+  const { id } = useParams();
+
+  const poaRequest =
+    poaRequests && poaRequests.find(r => r.id === Number(id))?.attributes;
```

Comment:
Because the initial state returned by `usePOARequests` is now an array, we don't need to first check that `poaRequests` is falsy, so this should be:
```javascript
const poaRequest = poaRequests.find(r => r.id === Number(id))?.attributes;
```